doc/guides/building-node-with-ninja.md

```diff
@@ -0,0 +1,41 @@
+# Building Node with Ninja
+
+The purpose of this guide is to show how to build Node.js using [Ninja][], as doing so can be much quicker than using `make`. Please see [Ninja's site][Ninja] for installation instructions (unix only).
+
+To build Node with ninja, there are 4 steps that must be taken:
+1. Configure the project via `./configure` as usual.
+2. Make `tools/gyp_node.py` produce Ninja output.
+3. Run Ninja in `out/Release` to produce release output.
+4. Link `out/Release/node` to `./node`.
+
+Those 4 steps can be achieved with the following commands:
+1. `./configure` (regular configure based on OS)
+2. `tools/gyp_node.py -f ninja` (passes `-f ninja` to `gyp`)
+3. `ninja -C out/Release` (run `ninja` within the `out/Release` directory, where ninja instructions were put)
+4. `ln -fs out/Release/node node` (create a link to `./node`, using `-f` to overwrite and `-s` to make a symlink)
+
+When running `ninja -C out/Release` you will see similar output if the build has succeeded:
```

Comment:
Similar to what? This should probably be `s/similar output/output similar to the following/`